app/src/main/java/com/mashup/mobalmobal/customview/DonatorView.kt

```diff
@@ -0,0 +1,51 @@
+package com.mashup.mobalmobal.customview
+
+import android.content.Context
+import android.util.AttributeSet
+import android.util.Log
+import android.view.LayoutInflater
+import android.widget.ImageView
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.mashup.base.image.GlideRequests
+import com.mashup.mobalmobal.R
+import com.mashup.mobalmobal.databinding.ViewDonationBinding
+import com.mashup.mobalmobal.databinding.ViewDonatorBinding
+
+class DonatorView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null
+) : ConstraintLayout(context, attrs){
+
+    private var binding: ViewDonatorBinding=
+        ViewDonatorBinding.inflate(LayoutInflater.from(context), this)
+
+
+    private val profileImageViews: List<ImageView> by lazy {
+        listOf(
+            binding.ivProfileFirst,
+            binding.ivProfileSecond,
+            binding.ivProfileThird
+        )
+    }
+
+    fun setDonatorProfiles(glideRequests: GlideRequests, profiles: List<String>) {
+        if(profiles.isNullOrEmpty()) return
```

Comment:
```suggestion
        if(profiles.isEmpty()) return
```
non-null하게 받아서 null체크는 필요 없어 보입니다

Reply:
수정했어요~